### PR TITLE
Implementation of moisture sensor with LoPy4 for both Pytrack and expansion board

### PR DESCRIPTION
**Pull request by kylecjo**

### CODE REVIEW

File: decoder.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    
+    var lat = bytes[8] + bytes[9] * 256 + bytes[10] * 65536 + bytes[11] * 4294967296;
+    var long = bytes[12] + bytes[13] * 256  + bytes[14] * 65536  + bytes[15] * 4294967296;
+    
+    decoded.lat = lat / 100000;
+    decoded.long = long / -10000;
+    
+    
+    return decoded;
```
Comment (LiyaoJiang1998) [inline decoder.js]:
```suggestion
    return{
        state: {
            deviceId: decoded.deviceId, 
            rawMoisture : decoded.rawMoisture,
            moistureDescription : decoded.moistureDescription,
            lat : decoded.lat,
            long : decoded.long
        }
    };
```

File: pytrack/main.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    return power
+
+
+def setup_single_lora_channel(lora):
+    # remove all the channels
+    for channel in range(0, 72):
+        lora.remove_channel(channel)
+    # set all channels to the same frequency
+    for channel in range(0, 8):
+        lora.add_channel(channel, frequency=903100000, dr_min=0, dr_max=3)
```
Comment (LiyaoJiang1998) [inline pytrack/main.py]:
```suggestion
        lora.add_channel(channel, frequency=FREQ, dr_min=0, dr_max=3)
```

File: expansionboard/main.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+def read_sensor(sensor, power_pin):
+    # take multiple readings and take the average to get a more reliable reading
+    print('reading sensor')
+    READING_DELAY_IN_S = 1
+    NUM_READINGS = 10
+
+    total = 0
+
+    for i in range(0, NUM_READINGS):
+        power_pin.value(1)
```
Comment (LiyaoJiang1998) [inline expansionboard/main.py:77]: line 77 and 82 is for pytrack only

File: expansionboard/main.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    return power
+
+
+def setup_single_lora_channel(lora):
+    # remove all the channels
+    for channel in range(0, 72):
+        lora.remove_channel(channel)
+    # set all channels to the same frequency
+    for channel in range(0, 8):
+        lora.add_channel(channel, frequency=903100000, dr_min=0, dr_max=3)
```
Comment (LiyaoJiang1998) [inline expansionboard/main.py]:
```suggestion
        lora.add_channel(channel, frequency=FREQ , dr_min=0, dr_max=3)
```

Comment (kylecjo) [reply expansionboard/main.py]: addressed in new commits

File: expansionboard/main.py (hunk 3 above)
Comment (kylecjo) [reply expansionboard/main.py:77]: addressed in new commits

File: pytrack/main.py (hunk 2 above)
Comment (kylecjo) [reply pytrack/main.py]: addressed in new commits

File: decoder.js (hunk 1 above)
Comment (kylecjo) [reply decoder.js]: addressed in new commits

**Pull request merged**